docs/adapters/index.md

```diff
@@ -12,6 +12,7 @@ These adapters are included in the Juttle runtime and can be used in Juttle prog
 
 * [file](../adapters/file.md)
 * [http](../adapters/http.md)
+* [io](../adapters/io.md)
```

Comment:
Yea, my other thought was using `terminal` (which may not actually be technically accurate) for the adapter name. Wasn't particularly fond of `io` or `terminal`, but thought both the `read` and `write` should be in the same adapter...perhaps not.

Could the adapter be called `stream`?
```
read stream -name "STDIN"
| (
  filter error != null | write stream -name "STDERR";
  filter error == null | write stream -name "STDOUT"
)
```
Meh...a bit more verbose.